docs/install-machine.md

```diff
@@ -57,6 +57,27 @@ to your PATH.
         $ docker-machine -v
         machine version 0.5.0 (3e06852)
 
+## Installing bash completion scripts
+
+The Machine repository supplies several `bash` scripts that add features such
+as:
+
+* command completion
+* a function that displays the active machine in your shell prompt
+* a function wrapper that adds a `docker-machine use` subcommand
+
+The `docker-machine` function executes `eval "$(docker-machine env X)"` in the
```

Comment:
@nathanleclaire With your recommended change above, I think this line can go away entirely.